Makefile

```diff
@@ -237,3 +237,31 @@ upload-sample-transfer:
 workflowcheck: # @HELP Detect non-determinism in workflow functions.
 workflowcheck: $(WORKFLOWCHECK)
 	workflowcheck ./...
+
+cluster-start: # @HELP Start cluster.
+	k3d cluster start sdps-local
+	tilt up
+
+cluster-stop: # @HELP Stop cluster.
+	k3d cluster stop sdps-local
+	tilt down
```

Comment:
`tilt down` is not going to work with the cluster stopped. I also like to keep the cluster as it was between restarts, so I rarely run `tilt down` in general.